### DISCUSSION TITLE
What is this error?

### DISCUSSION THREAD

**Discussion by 909sa** [dftbplus/dftbplus#1530] [General]
***  Geometry step: 214

 iSCC Total electronic   Diff electronic      SCC error    
Scalapack operation failed!
ppotrf in scalafx_ppotrf_dcomplex
Info: 65

**Comment by aradi** [comment]
The Cholesky-decomposition of the overlap matrix failed, as it is probably not positive definite any more. There is a fair chance, that you have a very unphysical geometry, check the position of the atoms, also with respect of the periodic boundary conditions.
